Disable some little-used format (broot is not an image editor ffs).
This saves 0.6 MiB.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42 +42 @@
-image = "0.24"
+image = { version = "0.24", default-features = false, features = ["gif", "jpeg", "ico", "png", "farbfeld", "jpeg_rayon"] }
